tests/fixed/example-class.php

```diff
@@ -0,0 +1,94 @@
+<?php
+namespace Example;
+
+use ArrayIterator;
+use Doctrine\Sniffs\Spacing\ControlStructureSniff;
+use Fancy\TestCase;
+use InvalidArgumentException;
+use IteratorAggregate;
+use const PHP_RELEASE_VERSION as PHP_PATCH_VERSION;
+use const PHP_VERSION;
+use function strlen as stringLength;
+use function substr;
+
+/**
+ * Description
+ */
+class Example implements IteratorAggregate
+{
+    private const VERSION = PHP_VERSION - (PHP_MINOR_VERSION * 100) - PHP_PATCH_VERSION;
+
+    /**
+     * @var int|null
+     */
+    private $foo;
+
+    /**
+     * @var string[]
+     */
+    private $bar;
+
+    /**
+     * @var bool
+     */
+    private $baz;
+
+    /**
+     * @var ControlStructureSniff|int|string|null
+     */
+    private $baxBax;
+
+    public function __construct(int $foo = null, array $bar = [], bool $baz = false, $baxBax = 'unused')
+    {
+        $this->foo    = $foo;
+        $this->bar    = $bar;
+        $this->baz    = $baz;
+        $this->baxBax = $baxBax;
+    }
+
+    /**
+     * Description
+     */
+    public function getFoo(): ?int
+    {
+        return $this->foo;
+    }
+
+
+    /**
+     * @return iterable
+     */
+    public function getIterator(): array
+    {
+        assert(null !== $this->bar);
+        return new ArrayIterator($this->bar);
```

Comment:
Do we already have a rule for insert a new line before control structures?

Reply:
No, we don't. As soon as this PR gets merged, I'll open a PR for that.